packages/react-debug-tools/src/__tests__/ReactHooksInspection-test.js

```diff
@@ -277,9 +277,17 @@ describe('ReactHooksInspection', () => {
     };
 
     expect(() => {
-      expect(() => {
+      // mock the Error constructor to check the internal of the error instance
+      try {
         ReactDebugTools.inspectHooks(Foo, {}, FakeDispatcherRef);
-      }).toThrow("Cannot read property 'useState' of null");
+      } catch (error) {
+        expect(error.message).toBe('Error rendering inspected component');
+        // error.cause is the original error
+        expect(error.cause).toBeInstanceOf(Error);
+        expect(error.cause.message).toBe(
+          "Cannot read property 'useState' of null",
+        );
+      }
```

Comment:
This testing approach looks good 👍🏼 but one small suggestion: add a verification step that an error was actually caught and examined. (Otherwise this test might create a false positive– if no error was thrown at all).

Reply:
That's right! Thanks!